source/logrepl/snapshot_test.go

```diff
@@ -0,0 +1,87 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logrepl
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/conduitio/conduit-connector-postgres/test"
+	sdk "github.com/conduitio/conduit-connector-sdk"
+
+	"github.com/matryer/is"
+)
+
+func TestLifecycle(t *testing.T) {
+	i := is.New(t)
```

Comment:
Nitpick: by convention we use `is := is.New(t)`, I know we are overwriting the package name `is` but after this call we actually don't have a use for the package anymore.